pkg/serverless/daemon/daemon.go

```diff
@@ -134,6 +134,18 @@ func StartDaemon(addr string) *Daemon {
 	return daemon
 }
 
+func wrapOtlpError(handle http.Handler) http.Handler {
```

Comment:
Since otlp.IsEnabled is global, would it be cleaner to add this check to the individual handlers? Or alternatively, just log the message once on startup?

Reply:
Good question. First off, as far as I know, we cannot tell at startup if the dd tracing layer is in use, meaning we'll have to log something at runtime.

I ultimately went with this solution for a couple reasons:
1. Avoids the `if otlp.IsEnabled()` call for every request, only log the message when we need to log the message.
2. We would need to add the log message in at least two locations, one at route `/hello` and the other at `/start-invocation`, to ensure we capture both universal and non-universal instrumented tracers. So, instead of writing the same message in two places, this solution allows us to only write it once.
3. This error message isn't exactly helpful to understanding how these routes work. Therefore, including the error message in the handler itself gums things up a bit. I can imagine a situation where someone sees this error, does a google search on it, then comes to this line of code. Once they read the comment, they'd know what they need to do to solve the problem.